package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "zc-main",
-  "version": "1.0.0",
+  "version": "v1.18.0",
   "author": "Zurichat",
```

Comment:
Why edit this? Leave it as it is. it should be `"version": "1.0.0",`. editing this will affect all other contributors so ensure youre only editing files that are related to your task